### DISCUSSION TITLE
Kirby CMS

### DISCUSSION THREAD

**Discussion by rickybass** [tailwindlabs/tailwindcss#1979] [General]
I am looking for some help implementing Kirby CMS with Tailwind. I am unsure how to do this and cannot find a tutorial and cannot seem to figure out the build process so that the components end up in the correct place. Any resources would be greatly appreciated.

**Comment by merelinguist** [comment]
Never used Kirby but from a brief look at the docs, one thing you could do would be include Tailwind from the CDN using the instructions here: https://getkirby.com/docs/cookbook/templating/art-directed-blog-posts

**Comment by pedroborges** [comment]
@rickybass Here's a starterkit with Tailwind you use as a reference: https://github.com/francisceril/kirby-tailwind-starterkit

I also use it with Laravel Mix in Kirby projects.

**Comment by rickybass** [comment]
Thanks Pedro

Ricky Bass

Pat F. Bass III, MD, MS, MPH
Professor of Medicine and Pediatrics
LSU Health Shreveport
P: 318-675-6490
C: 318-573-9413
F: 318-675-8150





On Jul 10, 2020, at 7:28 PM, Pedro Borges <notifications@github.com<mailto:notifications@github.com>> wrote:

*EXTERNAL EMAIL: EVALUATE*


@rickybass<https://nam01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Frickybass&data=02%7C01%7Cpbassi%40lsuhsc.edu%7Ce80403fe2e154426f3f808d82531495f%7C3406368982d44e89a3281ab79cc58d9d%7C0%7C0%7C637300240905083512&sdata=8fM1jgMIK0cXi6ez%2BpRQXpXFpTGHnrN86Pp3sp0K%2Bd0%3D&reserved=0> Here's a starterkit with Tailwind you use as a reference: https://github.com/francisceril/kirby-tailwind-starterkit<https://nam01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Ffrancisceril%2Fkirby-tailwind-starterkit&data=02%7C01%7Cpbassi%40lsuhsc.edu%7Ce80403fe2e154426f3f808d82531495f%7C3406368982d44e89a3281ab79cc58d9d%7C0%7C0%7C637300240905093504&sdata=By7oxQ4e6tnICM2pOOorNgUxxQrA5g%2FeR6oNCzFeNH4%3D&reserved=0>

I also use it with Laravel Mix in Kirby projects.
